test/test_rng.cpp

```diff
@@ -385,7 +387,7 @@ TEST_F(RNGTest, FunctionalUniform) {
       fusion->addOutput(tv1);
     }
 
-    auto second_offset = add(first_offset, IrBuilder::create<Val>(4L));
+    auto second_offset = add(first_offset, IrBuilder::create<Val>(1L));
```

Comment:
Please note that this is changed. In the past, we were dividing `4` for the user specified offset, which I don't think makes sense.

Reply:
Thanks for the explanation.